packages/rocketchat-ui/views/app/room.coffee

```diff
@@ -493,7 +493,40 @@ Template.room.onCreated ->
 	@autorun =>
 		@subscribe 'fullUserData', Session.get('showUserInfo'), 1
 
-	@subscribe 'roomModerators', @data._id
+	Meteor.call 'getRoomModeratorsAndOwners', @data._id, (error, results) ->
+		if error
+			return toastr.error error.reason
+
+		for record in results
+			RoomModeratorsAndOwners.insert record
+
+	RocketChat.callbacks.add('streamMessage', (msg) ->
```

Comment:
Sure, but I think you can move that code to some place that call only one time and prevent this processing, right? It's not necessary to execute this callback more than one time, or am I wrong?